tf2_tools/CMakeLists.txt

```diff
@@ -1,16 +1,12 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_tools)
 
-find_package(catkin REQUIRED COMPONENTS tf2
-                                         tf2_msgs
-                                         tf2_ros
-)
-
-catkin_package(
-   CATKIN_DEPENDS tf2
-                  tf2_msgs
-                  tf2_ros)
+find_package(ament_cmake REQUIRED)
 
-install(PROGRAMS scripts/view_frames.py
-        DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+install(
+  PROGRAMS
+  scripts/view_frames.py
+  DESTINATION lib/${PROJECT_NAME}
 )
+
+ament_package()
```

Comment:
Maybe enable the common linters for this package?

Reply:
This was actually a bad advice. This will lead the more differences between the upstream repo and this fork which makes porting changes between them more effort. Please revert the added checks and the style updates - sorry for the hassle.